src/Carnac.Tests/MessageProviderFacts.cs

```diff
@@ -61,12 +67,12 @@ public void recognises_shortcuts()
         }
 
         [Fact]
-        public void does_not_show_shortcut_name_on_partial_match()
+        public void does_not_show_key_press_on_partial_match()
```

Comment:
This is the test I changed the behaviour of.

Before if I was in visual studio and pressed `ctrl +r` that would appear in carnac right away. But because it is part of a chord shortcut I thought it might make sense to not show it until you completed/broke the chord.

What do you think @shiftkey